docs/fa2_parameters.md

```diff
@@ -99,6 +99,43 @@ You can then add as many transactions as you like to be processed by the contrac
 
 > Note: the properties holding the addresses of the spender and the recipient both end with an underscore: **from_** and **to_**.
 
+## The balance_of entrypoint
+Here is the type signature for the entrypoint parameter in Michelson:
+```
+(pair %balance_of
+  (list %requests
+    (pair
+      (address %owner)
+      (nat %token_id)
+    )
+  )
+  (contract %callback
+    (list
+      (pair
+        (pair %request
+          (address %owner)
+          (nat %token_id)
+        )
+        (nat %balance)
+      )
+    )
+  )
+)
+```
+This means that the entrypoint takes a pair annotated as *%balance_of*. On the left side of *%balance_of* pair takes a list of *%requests* structured as a pair of address as *%owner* and nat as *%token_id*. On the right side of *%balance_of* pair takes a contract annotated as *%callback* which the contract entrypoint type signature will be a list of pairs on the left side of the pair as *%request* that constructs with an address as *%owner and nat as *%token_id* and on the right side as nat as *%balance*
```

Comment:
small nitpick on punctuation, could we add a a comma (,) at this part:

> On the left side of `%balance_of`, `pair` takes a list of `%requests` structured as pair of....

and maybe we could use backtick (`) to denote code instead of italic for the annotations? That's just a stylistic choice from me though.

Reply:
i like the idea too, will update